src/reducers/quizReducer.js

```diff
@@ -6,6 +6,34 @@ export const quizReducer = (quizState, { type, payload }) => {
     case "SET_QUIZZES":
       return { ...quizState, quizList: payload };
 
+    case "SET_CURRENT_QUESTION_INDEX":
+      return { ...quizState, currentQuestionIndex: payload };
+
+    case "SET_SELECTED_OPTION":
+      return {
+        ...quizState,
+        quizList: quizState.quizList.map((quiz) =>
```

Comment:
suggestion: put a comment explaining what is happening in this case

Reply:
done